src/meta/js.js

```diff
@@ -47,7 +47,9 @@ async function linkModules() {
 		if (stats.isDirectory()) {
 			await file.linkDirs(srcPath, destPath, true);
 		} else {
-			await fs.promises.copyFile(srcPath, destPath);
+			// Instead of copying file, create a new file re-exporting it
+			// This way, imports in modules are resolved correctly
+			await fs.promises.writeFile(destPath, `export * from '${srcPath}'`);
```

Comment:
Seems to break on windows, the output was 

`export * from 'D:\github\NodeBB\node_modules\nodebb-plugin-custom-pages\static\lib\admin.js'`.

I tried
```
const posixPath = path.posix.resolve(__dirname, '../', modules[relPath].split(path.sep).join(path.posix.sep));
await fs.promises.writeFile(destPath, `export * from '${posixPath}'`);
```

Which outputs 

`export * from '/github/NodeBB/node_modules/nodebb-plugin-custom-pages/static/lib/admin.js'` 

That works on windows

Reply:
Hmm okay. I might just make the path relative to destpath.

Reply:
Alright should be fixed now - changed it to generate a posix-style relative path instead of an absolute path.